### DISCUSSION TITLE
ignore_throttled (starting elasticsearch server 7.16.2)

### DISCUSSION THREAD

**Discussion by cbb-colab** [dadoonet/fscrawler#1341] [Q&A] [answered]
I just installed fscrawler 2.8 (from docker image: dadoonet/fscrawler:noocr) against latest elasticsearch docker image (docker.elastic.co/elasticsearch/elasticsearch:7.16.2)

on indexing I got a warning which probably need to be changed in fscrawler.
Maybe a new config option could be setup here to change the behaviour for the newer elasticsearch  server in fscrawler ?

12:16:14,817 ESC[33mWARN ESC[m [o.e.c.RestClient] request [POST http://elasticsearch:9200/docs/_search?typed_keys=true&max_concurrent_shard_requests=5&ignore_unavailable=true&expand_wildcards=open&allow_no_indices=true&ignore_throttled=false&search_type=query_then_fetch&batched_reduce_size=512] returned 1 warnings: [299 Elasticsearch-7.16.2-2b937c44140b6559905130a8650c64dbd0879cfb **"[ignore_throttled] parameter is deprecated** because frozen indices have been deprecated. Consider cold or frozen tiers in place of frozen indices."]

**Comment by dadoonet** [comment, marked as answer]
Thanks for reporting. That's indeed caused by the current REST Client implementation.
The only thing we can do short term is to change the log level for this package but nothing more.

I will change the client implementation in the future.
